### PR TITLE
[printclient] - added information about primt of feature data

### PR DESCRIPTION
**Pull request by astroidex**
you can print featuredata from digiitizer feature or feature info. Documentation added to pritclient-element

### CODE REVIEW

File: de/functions/export/printclient.rst [hunk 1]
```diff
@@ -227,6 +227,104 @@ Ein Gruppen-abhängiger Druck könnte bei einer Gruppe namens "Gruppe 1" wie fol
...
+
+Definieren Sie im Drucktemplate ein Textfeld für die Informationen, die Sie für das selektierte Objekt ausdrucken möchten. Der Textfeldname hat immer den Prefix *feature.* gefolgt vom Namen der Spalte.
+
+.. code-block:: yaml
+
+  feature.name for column name of table poi
+
+
+2. Definieren Sie einen featureType und verweisen Sie auf das neue Drucktemplate in der config.yml
+--------------------------------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (astroidex) [reply de/functions/export/printclient.rst]: config.yml

**Comment by astroidex** [comment]
should be ok now.

File: en/functions/export/printclient.rst [hunk 2]
```diff
@@ -230,6 +230,100 @@ The print with a group named "Group 1" could look like this:
...
+          - template: a4_landscape_official_feature_data_demo
+            label: Demo with feature information print (landscape)
+
+
+3. Call feature print from featureInfo
+--------------------------------------
+
+Note: FeatureInfo is the information output from a OGC WMS service. It offers information for features at a click position.
+
+When you can configure a WMS you can generate a link with the following reference that will trigger the print with feature information.
```
Comment (rschwammborn) [inline en/functions/export/printclient.rst]:
```suggestion
If you configure a WMS, you can generate a link with the following reference that will trigger the print with feature information.
```

File: en/functions/export/printclient.rst [hunk 3]
```diff
@@ -230,6 +230,100 @@ The print with a group named "Group 1" could look like this:
...
+
+ <table>
+ <script src="http://code.jquery.com/jquery-latest.js"></script>
+ <tr>
+ <td class="th_quer">Print</td>
+ <td><a href="" onclick="parent.$('.mb-element-map').data('mapQuery').olMap.setCenter([[x],[y]]);parent.$('.mb-element-printclient:parent').data('mapbenderMbPrintClient').printDigitizerFeature('feature_demo',[gid]);parent.$('.mb-element-featureinfo:parent').data('mapbenderMbFeatureInfo').deactivate();return false">print feature information</a>
+ </td>
+ </tr>
+ </table>
+
```
Comment (rschwammborn) [inline en/functions/export/printclient.rst]: Auch hier müssen nochmal die Anmerkungen überprüft werden, die @AxxL für das MapServer-Beispiel in der deutschen Version gestellt hat.

File: en/functions/export/printclient.rst [hunk 4]
```diff
@@ -230,6 +230,100 @@ The print with a group named "Group 1" could look like this:
...
+.. code-block:: yaml
+    
+    printable: true
+
+When you click on the print button the print dialog is opens and offers the print templates that are defined for the feature type.
+
+Again you can choose the desired region and create a print pdf. The pdf will contain the information for the selected feature. 
+
+Note: The flexibility to move the print frame wont stop you from choosing a region that does not contain the feature that was selected. In this case the feature information does not match to the features that are displayed.
```
Comment (rschwammborn) [inline en/functions/export/printclient.rst]:
```suggestion
Note: The flexibility to move the print frame won't stop you from choosing a region that does not contain the feature that was selected. In this case the feature information does not match to the features that are displayed.
```

**Comment by astroidex** [comment]
Hello. The definition change in Mapbender version 3.0.8. So we have to redefine the example and modify the text

**Comment by fhaefker** [comment]
@astroidex 

please open a separated request for the correction

**Comment by astroidex** [comment]
ok. @CharlyUnicorn will do it

**Pull request merged**